### PR TITLE
Regen Docs

### PR DESCRIPTION
**Pull request by cdietrich**

### CODE REVIEW

File: org.eclipse.xtext.doc/contents/310_eclipse_support.html [hunk 1]
```diff
@@ -656,6 +656,42 @@ <h2 id="project-wizard">Project Wizard</h2>
...
+<p>Code minings come in two flavors: “header annotations” are printed in a separate line above the mined text, “inline annotations” are shown in-line. The following screenshot shows both flavors:</p>
+
+<p><img src="images/code_mining.png" alt="" /></p>
+
+<p>To enable code mining, the <code>org.eclipse.xtext.xtext.generator.ui.codemining.CodeMiningFragment</code> has to be integrated in the <code>language</code> section of the MWE2 file as follows:</p>
+
+<pre><code class="language-mwe2">fragment = org.eclipse.xtext.xtext.generator.ui.codemining.CodeMiningFragment {}
+</code></pre>
+
+<p>With execution of the generator a stub class <code>&lt;LanuageName&gt;CodeMiningProvider</code> is created in the <code>.codemining</code> sub-package of the UI plugin. Furthermore the provider is registered to the the <code>org.eclipse.ui.workbench.texteditor.codeMiningProviders</code> extension point in <code>plugin.xml</code>.</p>
```
Comment (ArneDeutsch) [inline org.eclipse.xtext.doc/contents/310_eclipse_support.html]: LanuageName   |   the the

Comment (cdietrich) [reply org.eclipse.xtext.doc/contents/310_eclipse_support.html]: then this needs to be fixed on the original code as well. will have a look

Comment (cdietrich) [reply org.eclipse.xtext.doc/contents/310_eclipse_support.html]: where is the the the i dont see it

Comment (ArneDeutsch) [reply org.eclipse.xtext.doc/contents/310_eclipse_support.html]: "Furthermore the provider is registered to the the"

**Pull request merged**